designer/client/src/components/graph/node-modal/aggregate/useAggParamsSerializer.tsx

```diff
@@ -46,7 +46,7 @@ export function useGroupByParamsSerializer(): [(text: string) => string[], (para
 
         switch (paramName) {
             case "groupBy":
-                return `{ ${content} }.toString`;
+                return `${content}`;
```

Comment:
IMO we should wrap keys in list if there is more than one key (now the widget works odd if we put multiple key)

Reply:
![Screenshot 2025-02-21 at 14 24 12](https://github.com/user-attachments/assets/0cf09b72-aae1-4faa-b178-248ea9d135d5)